charts/README.md

```diff
@@ -41,7 +41,7 @@ purposes). Use
 and
 
 ```shell
---set cloudserver-front.orbit.endpoint=https://dev.zenko.io
+--set cloudserver-front.orbit.endpoint=https://api.zenko.io
```

Comment:
Well, no, now it's not really in line with the doc, which refers to a *custom* endpoint.

Reply:
You are totally correct however this wasn't clear at first which is why I changed it (Since we are doing development and using orbit, it was thought that we were to use this endpoint based on the documentation). I'll add clarification in the README.

Also yes this ```orbit.endpoint``` not to be confused with ```endpoint``` that was changed in the other commits.